### PR TITLE
multiple pairs and change trade data to kline data

### PR DESCRIPTION
**Pull request by potatochick2020**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import reactLogo from './assets/react.svg'
 import './App.css'
 import Connection from './components/connection'
 
 function App() {
```
Comment (k9mil) [inline frontend/src/App.tsx]: pls put this file in another component/service/something.

Comment (k9mil) [reply frontend/src/App.tsx]: (as in stuff inside of this, as this should be for the home page component or w/e)

**Review by k9mil — CHANGES_REQUESTED** [review]
req

**Review by sabel8 — APPROVED** [review]
lgtm

**Pull request merged**